pygeoapi/api.py

```diff
@@ -48,6 +48,8 @@
 import uuid
 
 from dateutil.parser import parse as dateparse
+from lark.exceptions import UnexpectedInput
```

Comment:
Can we not include this?  And use a generic `except Exception as err` in the try/expect later on where this is used?

Reply:
Fixed by @ximenesuk